### DISCUSSION TITLE
how to create debian risc-v environment?

### DISCUSSION THREAD

**Discussion by kasperk81** [utmapp/UTM#4378] [UTM Help (macOS)]
according to https://wiki.debian.org/RISC-V#OS_.2F_filesystem_images, the riscv virt image ultimately comes from https://gitlab.com/giomasce/dqib. e.g. the latest artifacts (CI/CD > Jobs > select the latest riscv job > Browse): https://gitlab.com/giomasce/dqib/-/jobs/2977301018/artifacts/browse/artifacts/.

i am having a hard time to use those kernel/initd/.qcow2 artifact files with utm. can someone give a brief walkthrough how to get it working (i just need a console to test a few shell commands, don't mind the display if console has issues). starting with; should i select Virtual Machine or Emulate to use those debian artifacts? and what next..

**Comment by rxhfcy** [comment]
Try if the prebuilt RISC-V UTM image from [this GitHub issue](https://github.com/utmapp/mac.getutm.app/issues/31) works for you?
